### DISCUSSION TITLE
duplicate photo +1

### DISCUSSION THREAD

**Discussion by neoclassique** [immich-app/immich#4715] [Q&A] [answered]
Hello, I modified the quality of thumbnails and I have duplicate thumbnails, one of which is smaller, how can I resolve this problem?

edit: it's not because of the change in size of thumbnails

**Comment by jrasm91** [comment]
Can you clarify what you mean by this? When you change the path it should just overwrite the exist files. Regardless there are two thumbnails per asset, so it is expected to see two files per asset id on the filesystem.

**Reply by neoclassique** [reply, marked as answer]
I explained myself badly, I uploaded identical photos but of different sizes, immich upload both including one with ****+1.jpg, how to delete them?
I can delete them directly from the library folder but will this create problems with immlich?
